### PR TITLE
316 make the encryption optional

### PR DESCRIPTION
**Pull request by SRichner**
@casaout looking at this now, the emphasis on `(...single) *file was created* (based...)` seems a bit weird. What do you think?

image

### CODE REVIEW

**Comment by casaout** [comment]
> @casaout looking at this now, the emphasis on `(...single) *file was created* (based...)` seems a bit weird. What do you think?
> 
> image

I agree - should we call it `an export file was created` or `a data-file` (related to how we refer to it elsewhere)?

File: src/electron/src/views/DataExportView.vue [hunk 1]
```diff
@@ -219,8 +220,9 @@ function revealItemInFolder(event: Event) {
               <p>
                 Please click "Next" once you are ready to
                 <b class="dark:text-white">first review, and later share your data</b>. The export
-                that will be created with your permission in the next step will be encrypted and
-                password-protected.
+                that will be created with your permission in the next step will be
```
Comment (casaout) [inline src/electron/src/views/DataExportView.vue:223]: how will this read if dataExportEncrypted==false?

Isn't it:
> The export that will be created with your permission in the next step will be.

Should  we hide the entire sentence?

Comment (SRichner) [reply src/electron/src/views/DataExportView.vue]: good catch, updated so it hides the complete sentence

**Comment by SRichner** [comment]
> > @casaout looking at this now, the emphasis on `(...single) *file was created* (based...)` seems a bit weird. What do you think?
> > image
> 
> I agree - should we call it `an export file was created` or `a data-file` (related to how we refer to it elsewhere)?

How about `Your data was exported and we created a file based on your preferences on the previous page. To share this file with the researchers, please take the following steps:`?

**Review by casaout — APPROVED** [review]

**Pull request merged**